macros/commands/enforce_project_configurations.sql

```diff
@@ -0,0 +1,66 @@
+{%- macro enforce_project_configurations(enforce_owners=true, enforce_tags=false, enforce_meta_params=[], enforce_config_params=[], exclude_sources=false) -%}
```

Comment:
`enforce_meta_params` sounds like a boolean value.
What do you think about renaming to `required_meta_keys` or something in that area?
Same goes for the configuration.